createwindow/window_darwin.go

```diff
@@ -21,3 +22,20 @@ func ChromeCommand(layout Layout, userdatadir, display string, debuggingPort int
 	}
 	return cmd
 }
+
+func DisplayError(err error, cmdErrorHandler func(err error) error, isFatal bool) error {
+	icon :=  "caution"
+	if isFatal {
+		icon = "stop"
+	}
+
+	applescript := command.ExternalCommand{
+		Path: "osascript",
+		Arg: []string{"-e",fmt.Sprintf("display dialog \"%s\" with icon %s", err, icon)},
```

Comment:
If your err contains a double quote, this will fail. I think this should work better:

fmt.Sprintf("display dialog %q with icon %s", err.Error(), icon)

This is also a great example of why having tests is necessary. If you separate out the Arg[] creation in a separate function, you can (and should test it). So please do.

Reply:
Tests pending.